packages/mason/test/src/mason_yaml_test.dart

```diff
@@ -76,6 +76,15 @@ void main() {
             .having((g) => g.path, 'path', instance.git!.path),
       );
     });
+
+    test('isLocal', () {
+      // ignore: avoid_redundant_argument_values
+      final remoteInstance = BrickLocation(version: 'any');
```

Comment:
minor but can we add a check for a git based brick location?

Reply:
Totally forgot that, thanx